### ISSUE TITLE
Runtime error when installing in gVim Portable on Windows 7

### ISSUE THREAD

**Issue by thdoan** [VundleVim/Vundle.vim#735]
Steps to reproduce:

1. Unzip gVim Portable to any folder (let's assume `D:\portable\gVim` here)
2. Clone Vundle to `D:\portable\gVim\Data\settings\vimfiles\bundle\Vundle.vim`
3. Add the following to the end of `D:\portable\gVim\Data\settings\_vimrc`:

    ```
    filetype off  " Required

    " Set the runtime path to include Vundle and initialize
    set rtp+=$VIMRUNTIME/../../../Data/settings/vimfiles/bundle/Vundle.vim
    call vundle#begin('$VIMRUNTIME/../../../Data/settings/vimfiles/bundle')

    " Let Vundle manage Vundle, required
    Plugin 'gmarik/Vundle.vim'

    " Add all your plugins here (note older versions of Vundle used Bundle instead of Plugin)

    " All of your Plugins must be added before the following line
    call vundle#end()          " Required
    filetype plugin indent on  " Required
    ```

4. Start gVim and execute `:PluginInstall`

I get the following error dialog:

![image](https://cloud.githubusercontent.com/assets/4110567/15923748/4091bc5e-2e61-11e6-8a53-585f96754447.png)

**Comment by arturogarza** [comment]
How did you clone Vundle to your portable drive? I'm trying to do the same thing so that I can run install Vundle on my portable Vim that I have installed on my USB. I also have portable Git. I have been trying forever to figure this out and nothing seems to work. I tried installing the portable MySysGit and still haven't been able to get it to work.

**Comment by thdoan** [comment]
@arturogarza I used Git shell and don't recall doing anything special. For example, to clone to `D:\portable` I do `cd /d/portable` followed by `git clone some_repo`.

**Comment by arturogarza** [comment]
@thdoan Hey, I finally ended up being able to install Vundle. I just got the Vundle "autoload" folder and inserted it's files "vundle and vundle.vim" into the .vim folder that I created under my gVimPortable files. It looks like this gvimPortable/Data/settings/.vim/bundle/Vundle.vim/autoload. The folder "vundle and file vundle.vim" are both in the "autoload" folder. 

I then opened up gVim and I used the :PluginInstall and it worked. Finally.

I used rtp+=~/.vim/bundle/Vundle.vim and Plugin 'VundleVim/vundle.vim'

It is a little more confusing on Windows to clone from Git as most of the clone code is meant for Linux. We have to manually go in and change stuff. 

I tried to install Fugitive using the same process however I keep getting an install error. Any ideas? Do you have Fugitive?

**Comment by thdoan** [comment]
@arturogarza No, I've never tried Fugitive, but after reading its readme it's something I should probably try :).

**Comment by arturogarza** [comment]
@thdoan Let me know if you get it working?
